server/fixtures/category.js

```diff
@@ -0,0 +1,4 @@
+const newCategory = {
+  categoryName: 'default'
```

Comment:
It's already under category table do I need to know explicitly that it's a categoryName.
When I want to use it somewhere it will be `newCategory.categoryName`. I'd prefer. `category.name`